hacktheback/rest/forms/views/walkin.py

```diff
@@ -0,0 +1,43 @@
+from rest_framework import generics
+from rest_framework.exceptions import NotFound, ValidationError
+from rest_framework.permissions import IsAdminUser
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from hacktheback.account.models import User
+from hacktheback.forms.models import HackathonApplicant
+from hacktheback.rest.permissions import AdminSiteModelPermissions
+
+from ....forms.utils import send_rsvp_email
+
+
+class WalkInAdmissionAPIView(generics.GenericAPIView):
+    permission_classes = (AdminSiteModelPermissions,)
+    queryset = HackathonApplicant.objects.all()
+
+    def post(self, request, *args, **kwargs) -> Response:
+        assert "email" in request.data
+        email = request.data.get("email")
+        try:
+            applicant = HackathonApplicant.objects.get(application__user__email=email)
+        except HackathonApplicant.DoesNotExist as not_found:
+            raise NotFound(detail="Applicant does not exist") from not_found
+        try:
+            user = User.objects.get(email=email)
+        except HackathonApplicant.DoesNotExist as not_found:
+            raise NotFound(detail="Applicant does not exist") from not_found
```

Comment:
this except needs to catch User.DoesNotExist, not HackathonApplicant.DoesNotExist